scalatest/src/main/scala/org/scalatest/tools/Framework.scala

```diff
@@ -430,16 +431,17 @@ class Framework extends SbtFramework {
           annotationClass.getName
         else
           value
-      }
+      }).asInstanceOf[Array[String | Null]]
     
-    def execute(eventHandler: EventHandler, loggers: Array[Logger]) = {
+    def execute(eventHandler: EventHandler | Null, _loggers: Array[Logger | Null] | Null) = {
+      val loggers = _loggers.nn.map(_.nn)
       if (accessible || runnable) {
         val suite =
           try {
             if (runnable) { // When it is runnable WrapWith is available, this will take precedence and this behavior will be consistent with Runner and the old ScalaTestFramework.
               val wrapWithAnnotation = suiteClass.getAnnotation(classOf[WrapWith])
               val suiteClazz = wrapWithAnnotation.value
-              val constructorList = suiteClazz.getDeclaredConstructors()
+              val constructorList : Array[Constructor[_]] = suiteClazz.getDeclaredConstructors().map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.